kontonummer.js

```diff
@@ -432,20 +438,26 @@
     var validateChecksum = function(bank, bankNumber) {
         var bank_name = null;
         var errors = [];
+        var warnings = [];
         var ctrlNum = getCtrlNum(bank.lengths.control, bankNumber);
 
         if (bank.regex.test(bankNumber)) {
             bank_name = bank.name;
             if (!(bank.modulo === 11 && mod11(ctrlNum)) && !(bank.modulo === 10 && mod10(ctrlNum))) {
-                errors.push('bad_checksum');
+                if(bank.warnOnBadChecksum) {
```

Comment:
We don't have linting on this repo yet (since it was forked), but let's try to keep consistent:

```suggestion
                if (bank.warnOnBadChecksum) {
```